OpenAI/Packages/com.openai.unity/Runtime/Models/Model.cs

```diff
@@ -81,42 +87,47 @@ public Model(string id)
         /// <summary>
         /// More capable than any GPT-3.5 model, able to do more complex tasks, and optimized for chat. Will be updated with our latest model iteration.
         /// </summary>
-        public static Model GPT4 { get; } = new Model("gpt-4") { OwnedBy = "openai" };
+        public static Model GPT4 { get; } = new Model("gpt-4", "openai");
```

Comment:
```suggestion
        public static Model GPT4 { get; } = new("gpt-4", "openai");
```

Reply:
Cool, I wanted to do this but wasn't sure if it would be ok.